x-pack/plugins/session_view/public/components/Process/index.tsx

```diff
@@ -200,23 +205,23 @@ function Process({ process, isSessionLeader = false, depth = 0, onProcessSelecte
   }
 
   // TODO: not customizable for now (cmd previously offered a template string to render)
-  const template = (process: IProcess) => {
-    const event = process.getLatest();
-    const { args, working_directory } = event.process;
-    const { searchMatched } = process;
+  const template = (templateProcess: IProcess) => {
```

Comment:
why the var rename?

Reply:
Eslint was complaining saying:

`'template' is already declared in the upper scope.eslint@typescript-eslint/no-shadow`

this rule is to avoid ambiguity, and the declaration was in the function itself, although, maybe we can change the function or the parameter to a better name and still avoid the ambiguity

Reply:
ah yea that makes sense. we could probably just ditch the function var as I don't think i'll need to pass it through (it did in the old code for some other convoluted reasons.. so i'm fine to remove the function var and just ref the process var in the closure.

Reply:
Yeah I agree to remove the function `variable` and stick with the parent `variable`

Reply:
pushed this change